82-Screen-listado-de-testimonios

ByMofm opened the pull request:
-Se creo el componente /Backoffice/Testimonials

-Se añadieron las rutas para renderizarlo dentro del app.js

![testimonials1](https://user-images.githubusercontent.com/85798648/167875416-c031ccc7-fac3-4f95-b73e-2748803fe377.png)
s

Commit:
First Commit

-Se creo el componente /Backoffice/Testimonials

-Se añadieron las rutas para renderizarlo dentro del app.js

Galaeno commented:
@ByMofm agregar id de jira al titulo del pr

Galaeno commented:
@ByMofm tenés conflictos

ByMofm committed:
Merge branch 'main' into 82-Screen-listado-de-testimonios

ByMofm commented:
Corregida ambas cosas.

Galaeno commented:
@ByMofm tenés conflictos

ByMofm committed:
Merge branch 'main' into 82-Screen-listado-de-testimonios

ByMofm commented:
Corregido.

Galaeno commented on src/App.js:
```diff
@@ -17,11 +17,13 @@ import { LayoutBackOffice } from './layout/LayoutBackOffice';
 import EditHome from './components/EditHome/EditHome';
 import { EditUserForm } from './components/editUserForm/EditUserForm';
 import UserProfile from './components/Profile/UserProfile';
+import OutletLayout from './layout/OutletLayout';
```
ÉSto ya fue importado en la linea 4

Galaeno commented on src/components/backoffice/testimonials.js:
```diff
@@ -0,0 +1,90 @@
+
+  const loadValues = async () => {
+    const response = await get('/testimonials');
+    if (response) {
```
Si hay un error?, estaría bueno mostrar un alerta de error

ByMofm replied:
Corregido

Commit:
update

-Se modifico un import repetido dentro del app.js
-Se modifico el componente testimonials cambiándolo a un componente de tipo jsx
-Se destructurizo el request para poder  tirar un alert del posible error usando un servicio ya creado.

Galaeno commented on src/App.js:
```diff
@@ -45,12 +45,12 @@ function App() {
         </Route>
         {/* Menu de opciones BackOffice User */}
         <Route path="/backoffice/user" element={<UserProfile />} />
-        <Route path="/backoffice/news/create" element={<FormNews />} />
```
Por qué se borró?

ByMofm replied:
Corregido, no se deberían borrar.

Commit:
Update App

-Se volvieron a agregar unas rutas eliminadas por error

ByMofm committed:
Merge branch 'main' into 82-Screen-listado-de-testimonios

Galaeno commented:
@ByMofm tenés conflictos

ByMofm committed:
Merge branch 'main' into 82-Screen-listado-de-testimonios

ByMofm commented:
Solucionado.
